src/components/CommentList/CommentList.scss

```diff
@@ -1 +1,7 @@
-//
```

Comment:
Replace line comments with block comments for better readability and consistency 📝